identity-platform/gke/frontend/Dockerfile

```diff
@@ -16,8 +16,8 @@
 
 FROM python:3.8
 ENV PORT=8080
+WORKDIR /app
 COPY requirements.txt .
```

Comment:
Can we just `COPY . ./` here instead of copying twice?

Reply:
Maybe, but I'd argue that's best addressed in a different PR.

(This was supposed to be a "quick" lint fix. 😉 )

Reply:
Please remove from the PR